javaparser-core/src/main/java/com/github/javaparser/printer/PrettyPrinterConfiguration.java

```diff
@@ -44,6 +45,15 @@ public PrettyPrinterConfiguration setIndent(String indent) {
         return this;
     }
 
+    public boolean isNormalizeEolInComments() {
```

Comment:
maybe in this case the name should be `normalizeEolInComment` or `isEolInCommentsNormalizationEnabled`, not sure

Reply:
I think the first one is more consistent